os_command_injection/CODE-SECX_Vulnerable-codes/Command_Injection_1.js

```
@@ -0,0 +1,69 @@
+    eval(code); // Vulnerable
+});
+
+app.get('/run-dynamic', (req, res) => {
+    const code = req.query.code;
+    const dynamicFunction = new Function(code);
+    dynamicFunction(); // Vulnerable
+});
+
+process.env.PATH = process.env.PATH + ':/malicious/path'; // Vulnerable
+
+const { exec } = require('child_process');
```

Biome (2.1.2)

[error] Shouldn't redeclare 'exec'. Consider to delete it or rename it.

'exec' is defined here:


(lint/suspicious/noRedeclare)